fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -255,26 +255,62 @@ elif [ $DistortionCorrection = TOPUP ]; then
       --gdcoeffs=${GradientDistortionCoeffs} \
       --topupconfig=${TopupConfig}
 
+###########################################################################################################################################################################
+##Bene Changes: Use T2 instead of T1
+#if useT2=none fake T2 and make T2=T2wRestoreImage and apply T1 mask to make T2wRestoreImageBrain  
+#TODO 
+#Here is how I have been faking the T2 and it seems to be working better than using the T1 brain because of the high intensity eyes. 
+#${FSLDIR}/bin/fslmaths ${T1wFolder}/${T1wRestoreImage} -mul -1 ${T1wFolder}/FakeT2Head.nii.gz
+#${FSLDIR}/bin/fslmaths ${T1wFolder}/FakeT2Head.nii.gz -add 500 ${T1wFolder}/FakeT2Head.nii.gz
+#${FSLDIR}/bin/fslmaths ${T1wFolder}/FakeT2Head.nii.gz -uthrp 95 ${T1wFolder}/FakeT2Head.nii.gz
+#${FSLDIR}/bin/fslmaths ${T1wFolder}/FakeT2Head.nii.gz -mas ${T1wFolder}/T1w_acpc_brain_mask.nii.gz ${T1wFolder}/FakeT2brain.nii.gz
+#TODO define T2 images (need to make this better but hard code for now)
+echo "this has only been testin with T2, if we don't have one, fake it here"
+#Create fake "T2" if it doesn't exist
+if [ $useT2 = "True" ] ; then
+  T2wRestoreImage=T2w_acpc_dc_restore
+  T2wRestoreImageBrain=T2w_acpc_dc_restore_brain
+  T1wImage=T1w_acpc_dc_restore ##not sure where this comes from, so this could be a different image. 
+else
+  ${FSLDIR}/bin/fslmaths ${T1wFolder}/${T1wRestoreImage} -mul -1 ${T1wFolder}/T2w_acpc_dc_restore.nii.gz
+  ${FSLDIR}/bin/fslmaths ${T1wFolder}/T2w_acpc_dc_restore.nii.gz -add 500 ${T1wFolder}/T2w_acpc_dc_restore.nii.gz
+  ${FSLDIR}/bin/fslmaths ${T1wFolder}/T2w_acpc_dc_restore.nii.gz -uthrp 95 ${T1wFolder}/T2w_acpc_dc_restore.nii.gz
+  ${FSLDIR}/bin/fslmaths ${T1wFolder}/T2w_acpc_dc_restore.nii.gz -mas ${T1wFolder}/T1w_acpc_brain_mask.nii.gz ${T1wFolder}/T2w_acpc_dc_restore_brain.nii.gz
+  T2wRestoreImage=T2w_acpc_dc_restore
+  T2wRestoreImageBrain=T2w_acpc_dc_restore_brain
+  T1wImage=T1w_acpc_dc_restore ##not sure where this comes from, so this could be a different image.
+fi
```

Comment:
Since this new addition included the T2, I needed to figure out what to do if the subjects don't have a T2. This part is messy, but it was my way of dealing with subjects that don't have a T2, i.e. if they don't have a T2, we fake the T2 from the T1. Alternatively we could also just say T2w =T1w if useT2=false.

Reply:
Typo on line 268: "testin" s/b "tested".
At line 273,281: _please_ do not change the definition of T1wImage - it is used all over the place and should always refer to the same file (T1w_acpc_dc). There is already a definition for T1w_acpc_dc_restore - it's called T1wRestoreImage. Thanks!

Reply:
Note: if you tested with the changes you have made here, then the convertwarp at line 289 (in this version) will have been done with T1w_acpc_dc_restore  instead of T1w_acpc_dc. So, please decide which file it _should_ be using and put in any needed changes. That is, if you really _want_ it to use T1w_acpc_dc_restore, then change that line to use ${T1w_acpc_dc_restore}. THANKS!

Reply:
@b3n3dikt I concur with @kathy-snider here too.  Also, on line 270 above, is the `$useT2` variable supposed to be a string with a capitalized `"True"`?  That seems odd and out of place.  @kathy-snider doesn't BASH support Booleans of their own?  We should use those instead of oddly specific strings, but maybe this is a future bugfix.